modules/deploy/partials/linux/install-fips.adoc

```diff
@@ -0,0 +1,28 @@
+= Install Redpanda for FIPS compliance
+
+To run Redpanda in FIPS compliance mode, you must install the `redpanda-fips` package, which is separate from and dependent upon the `redpanda` install package already being installed. Install the following packages after installing `redpanda`:
+
+`redpanda-fips`
+
+- Contains the OpenSSL FIPS-approved module and scripts required to set up and run Redpanda in FIPS-compliance mode.
+- Depends upon the successful installation of the `redpanda` package.
+- Includes the `fips.so` cryptographic provider (built from OpenSSL v3.0.9, which is the latest FIPS 140-2 approved module) and a copy of the OpenSSL application.
+- Executes `openssl fipsinstall` against the `fips.so` module, which generates a `fipsmodule.cnf` file that is used during the module’s POST (power-on-self-test) to validate the integrity of the module.
+
+`redpanda-rpk-fips`
+
+- Contains a version of `rpk` built with the https://github.com/microsoft/go[Microsoft GoLang compiler^] and https://github.com/microsoft/go-crypto-openssl[Microsoft’s Go Crypto OpenSSL package^] to which `rpk` is linked, and uses the FIPS-approved version of OpenSSL.
+
+To install Redpanda for FIPS compliance, run:
+
+[,bash]
+----
+sudo apt install -y redpanda-rpk-fips redpanda-tuners redpanda redpanda-fips
+----
+
```

Comment:
NOTE: Alternatively, you could run `sudo apt install -y redpanda-fips`, which also picks up and includes the `redpanda` install package.